CMakeLists.txt

```diff
@@ -155,7 +159,11 @@ if(${IREE_BUILD_COMPILER})
 endif()
 
 if(${IREE_BUILD_DEBUGGER} OR ${IREE_BUILD_SAMPLES})
+  # sdl2 logs are spammy - change log level while adding
```

Comment:
another way to do this is to nest the subdir in a function. That way it gets its own variable scope and you don't need to restore

```
function(include_sdl2)
  set(CMAKE_MESSAGE_LOG_LEVEL "WARNING")
  add_subdirectory(third_party....)
endfunction()
include_sdl2()
```

Reply:
Neat, done. (I think the explicit overriding/restoring is more readable, but this is fine too :P)